deltachat-ffi/deltachat.h

```diff
@@ -11,9 +11,6 @@ extern "C" {
 #endif
 
 
-#define DC_VERSION_STR "0.43.0"
```

Comment:
Lets not expose this string anymore. There is `dc_get_version_str()` that you can call.

Reply:
yip, we can remove it. it was handy in c-core to make sure to use a header compatible to the lib.a or lib.so ... i think this is no longer needed in rust :)